resources/themes/cura-light/icons/low/Cancel.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 10.5106L8.48936 7L7 8.48936L10.5106 12L7 15.5106L8.48936 17L12 13.4894L15.5106 17L17 15.5106L13.4894 12L17 8.48936L15.5106 7L12 10.5106Z" fill="#000E1A"/>
```

Comment:
The `clip-rule` property is not supported by Qt.
The `fill-rule` property is not used in this image since there are no self-intersections.
The `fill` property is not used in this image, since it is always recoloured. The colour shown here is incorrect (different from what we actually use in the interface).